### PR TITLE
Created the youtube clone page as the requirements

### PR DESCRIPTION
**Pull request by mateomh**
Created the fixed navbar with the logo, search bar and buttons.
Created the video display section with the title, view count, social buttons and creator information.
Created the sidebar with the playlist for the next videos.

### CODE REVIEW

**Review by expjazz — CHANGES_REQUESTED** [review]
## Hey @mateomh  and @rgcavalheiro 

#### The repo does not have a readme, so I need to mark it as invalid. I also noted that the numbers of commits were too low. I also advise using the word 'add' in the commits, like: add styling to the content section, instead of 'Style the content section, Start the playlist section'.


### Please make the required changes and resubmit for a new code review once you are ready
Good Luck and Happy Coding :pray:

P.S. If you have any questions about my review or need help, you can message me on slack (@ Expedito Andrade).:smile:

File: README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## Authors
+
+👤 **Mateo mojica**
+
+- Github: [@mateomh](https://github.com/mateomh)
+
+👤 **Rafael Cavalheiro**
+
+- Github: [@rgcavalheiro](https://github.com/githubhandle)
```
Comment (elmejdki) [inline README.md]: Please add also Linkedin and twitter profile links

File: README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# YOUTUBE CLONE PAGE
+
+> The objective of the project is to mimic a youtube video page with the navbar, the video display section and the sidebar with the upcomming videos. The base page taken was the current youtube video page .
+
+![screenshot](./screenshot.png)
```
Comment (elmejdki) [inline README.md]: you should use the screenshot of your clone, not the original website please change it

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 
## Great job so far guys :clap: but you need to work more on your clone in order to make it look better. :hammer: :wrench: :toolbox: 

- [x] you are missing a live demo link please add one, you can use [githack](https://raw.githack.com/) to generate a new link.

- [x] change the size of the search image

- [x] you are messing the comments section please add it.

- [x] you have a lot of space between the video and the sidebar please fix it.
image

- [x] align the title of the video with the video thumbnail 

### clone:
image

### original:
image

- [x] also use proper video titles and author names not just 'video title'

- [x] beneath the video you need to have the same title as in the original page, also the number of views and date should have grey color and smaller font-size:
image


- [x] Make this icons a little bit smaller:
image

- [x] align the channel image, title and subscribe button like in the original page:
### clone:
image

### original:
image

### Please make the required changes before requesting another code review.
## Happy Coding :tophat:

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi Team! 👋 
### Great job implementing the requested changes!! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****

 Good Luck and Happy Coding! 🍀 👩🏾‍💻

**Pull request merged**